src/components/molecules/ImageInput/ImageCollectionInput.tsx

```diff
@@ -108,7 +108,7 @@ export const ImageCollectionInput: React.FC<ImageInputProps> = (props) => {
           disabled={disabled}
           type="file"
           onChange={handleFileChange}
-          accept="image/png,image/x-png,image/gif,image/jpeg"
+          accept="image/png,image/x-png,image/gif,image/jpeg,image/webp"
```

Comment:
@margulies This (+ anywhere else across the platform that deals with uploading images) should be using `ACCEPTED_IMAGE_TYPES`

Reply:
fixed in 95a5b0761781b55a76026bd102f9dd080222675a

Reply:
that is the only place which seems to deal with image types